packages/rocketchat-lib/server/models/Rooms.js

```diff
@@ -88,6 +88,10 @@ class ModelRooms extends RocketChat.models._Base {
 
 	// FIND
 
+	findWithUsername(username, options) {
```

Comment:
@rodrigok we'll need to rewrite this on `no-cache` branch

Reply:
Yes